plugins/org.jkiss.dbeaver.ui/src/org/jkiss/dbeaver/ui/controls/folders/TabbedFolderList.java

```diff
@@ -1021,7 +988,7 @@ public void addSelectionListener(SelectionListener listener) {
 
     public void handleTraverse(TraverseEvent e) {
         if (e.detail == SWT.TRAVERSE_PAGE_PREVIOUS || e.detail == SWT.TRAVERSE_PAGE_NEXT) {
-            if ((e.stateMask & SWT.ALT) != SWT.ALT) {
+            if ((e.stateMask & SWT.SHIFT) != SWT.SHIFT) {
```

Comment:
Is it necessary to change shortcuts?

Reply:
Have you checked it in NVDA? I guess you should as it may stop work there by the similar reason

Reply:
Checked

Reply:
Ok, but the comment is still
// Only in case of CTRL+ALT+PG